web/ui/react-app/src/pages/rules/RulesContent.tsx

```diff
@@ -95,7 +95,7 @@ export const RulesContent: FC<RouteComponentProps & RulesContentProps> = ({ resp
                             <strong>for:</strong> {formatRange(r.duration)}
                           </div>
                         )}
-                        {Object.entries(r.labels).length > 0 && (
+                        {r.labels && Object.entries(r.labels).length > 0 && (
                           <div>
                             <strong>labels:</strong>
                             {Object.entries(r.labels).map(([key, value]) => (
```

Comment:
This is a pre-existing condition, but in case you sort out the quoting/escaping in the other PR, it would be good to do the same here with the label values.

Reply:
I will do both at the same time